pkgs/applications/misc/linux-notification-center/default.nix

```diff
@@ -0,0 +1,46 @@
+{ stdenv, lib, fetchurl, autoPatchelfHook 
+, gtk3, gobject-introspection, libxml2 }:
+
+let
+  dbusService = fetchurl {
+    url = "https://github.com/phuhl/linux_notification_center/raw/master/com.ph-uhl.deadd.notification.service.in";
+    sha256 = "0jvmi1c98hm8x1x7kw9ws0nbf4y56yy44c3bqm6rjj4lrm89l83s";
+  };
+in 
+stdenv.mkDerivation rec {
+
+  pname = "linux_notification_center";
+  version = "1.7.2";
+
+  src = fetchurl {
+    url = "https://github.com/phuhl/linux_notification_center/releases/download/${version}/deadd-notification-center";
+    sha256 = "13f15slkjiw2n5dnqj13dprhqm3nf1k11jqaqda379yhgygyp9zm";
+  };
+
+  dontUnpack = true;
+
+  nativeBuildInputs = [ autoPatchelfHook ];
+
+  buildInputs = [
+    gtk3
+    gobject-introspection
+    libxml2
+  ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    cp $src $out/bin/deadd-notification-center
+    chmod +x $out/bin/deadd-notification-center
+
+    mkdir -p $out/share/dbus-1/services
+    sed "s|##PREFIX##|$out|g" ${dbusService} > $out/share/dbus-1/services/com.ph-uhl.deadd.notification.service 
+  '';
+
+  meta = with lib; {
+    description = "A notification daemon/center for linux";
+    homepage = https://github.com/phuhl/linux_notification_center;
+    license = licenses.bsd3;
+    maintainers = [ maintainers.pacman99 ];
+    platforms = platforms.linux;
+  };
+}
```

Comment:
```suggestion
{ stdenv
, fetchurl
, autoPatchelfHook
, gtk3
, gobject-introspection
, libxml2
}:

let
  version = "1.7.2";
  dbusService = fetchurl {
    url = "https://github.com/phuhl/linux_notification_center/raw/${version}/com.ph-uhl.deadd.notification.service.in";
    sha256 = "0jvmi1c98hm8x1x7kw9ws0nbf4y56yy44c3bqm6rjj4lrm89l83s";
  };
in
stdenv.mkDerivation rec {
  inherit version;
  pname = "linux-notification-center";

  src = fetchurl {
    url = "https://github.com/phuhl/linux_notification_center/releases/download/${version}/deadd-notification-center";
    sha256 = "13f15slkjiw2n5dnqj13dprhqm3nf1k11jqaqda379yhgygyp9zm";
  };

  dontUnpack = true;

  nativeBuildInputs = [ autoPatchelfHook ];

  buildInputs = [
    gtk3
    gobject-introspection
    libxml2
  ];

  installPhase = ''
    mkdir -p $out/bin $out/share/dbus-1/services
    cp $src $out/bin/deadd-notification-center
    chmod +x $out/bin/deadd-notification-center
    sed "s|##PREFIX##|$out|g" ${dbusService} > $out/share/dbus-1/services/com.ph-uhl.deadd.notification.service 
  '';

  meta = with stdenv.lib; {
    description = "A haskell-written notification center for users that like a desktop with style";
    homepage = "https://github.com/phuhl/linux_notification_center";
    license = licenses.bsd3;
    maintainers = [ maintainers.pacman99 ];
    platforms = platforms.linux;
  };
}
```

Here are a couple fixes for this current binary release version:

* version the `...service.in` file
* do both mkdir in one command
* just use `stdenv` dont also import `lib`
* quote the url in homepage

Reply:
Well the highlighting looks like crap :unamused: